presto-main/src/test/java/io/prestosql/sql/planner/iterative/rule/TestReorderJoins.java

```diff
@@ -96,8 +91,8 @@ public void testKeepsOutputSymbols()
                 .on(p ->
                         p.join(
                                 INNER,
-                                p.values(new PlanNodeId("valuesA"), ImmutableList.of(p.symbol("A1"), p.symbol("A2")), TWO_ROWS),
```

Comment:
Is that old `values(...)` method still used?

Reply:
yes, it provides implementation for other `values()` methods.